core-plugins/docs/ExcelInputReader-batchsource.md

```diff
@@ -78,26 +78,26 @@ generate structured records according to the inputs.
 
 The plugin JSON Representation will be:
 
-{
-  "name": "ExcelInputReader",
-  "type": "batchsource",
-  "properties": {
-        "filePath": "hdfs://<namenode-hostname>:9000/cdap",
-        "filePattern": ".*",
-        "memoryTableName": "inventory-memory-table",
-        "reprocess": "false",
-        "sheet": "Sheet1",
-        "sheetValue": "-1",
-        "columnList": "A,B",
-        "columnMapping": "B:name,C:age"
-        "skipFirstRow": "false",
-        "terminateIfEmptyRow": "false",
-        "rowsLimit": "" ,
-        "outputSchema": "A:string",
-        "ifErrorRecord" : "Ignore error and continue",
-        "errorDatasetName": ""
-   }
-}
+    {
+      "name": "Excel",
```

Comment:
This should be in the Markdown code block : https://daringfireball.net/projects/markdown/syntax#precode